src/CommonGLPI.php

```diff
@@ -1058,23 +1072,35 @@ class='btn btn-sm btn-icon btn-ghost-secondary'
                     'forceid' => $this instanceof CommonITILObject
                 ]);
                 echo "</h3>";
+            } else {
+                $twig_params = [
+                    'item'          => $this,
+                    'params'        => $options,
+                    'in_navheader'  => true
+                ];
+                if (method_exists($this, 'getFormHeaderToolbar')) {
+                    $header_toolbar = $this->getFormHeaderToolbar();
+                    // Do not trust return type will always be string
+                    if (is_array($header_toolbar)) {
+                        $twig_params['header_toolbar'] = $header_toolbar;
+                    }
+                }
```

Comment:
The proper way to ensure that method returns an array is probably to declare it in `CommonGLPI` with an empty implementation: `protected function getFormHeaderToolbar(): array { return []; }`.

```suggestion
                $twig_params['header_toolbar'] = $this->getFormHeaderToolbar();
```